src/main/java/io/supertokens/useridmapping/UserIdMapping.java

```diff
@@ -124,30 +126,25 @@ public static io.supertokens.pluginInterface.useridmapping.UserIdMapping getUser
             Main main, String userId,
             UserIdType userIdType)
             throws StorageQueryException {
-        try {
-            return getUserIdMapping(new TenantIdentifier(null, null, null), main, userId, userIdType);
-        } catch (TenantOrAppNotFoundException e) {
-            throw new IllegalStateException(e);
-        }
+        Storage storage = StorageLayer.getStorage(main);
+        return getUserIdMapping(new AppIdentifier(null, null, storage), userId, userIdType);
     }
 
-    public static boolean deleteUserIdMapping(TenantIdentifier tenantIdentifier, Main main, String userId,
+    public static boolean deleteUserIdMapping(AppIdentifier appIdentifier, String userId,
                                               UserIdType userIdType, boolean force)
-            throws StorageQueryException, ServletException, TenantOrAppNotFoundException {
-
-        UserIdMappingStorage storage = StorageLayer.getUserIdMappingStorage(tenantIdentifier, main);
+            throws StorageQueryException, ServletException {
 
         // referring to
         // https://docs.google.com/spreadsheets/d/17hYV32B0aDCeLnSxbZhfRN2Y9b0LC2xUF44vV88RNAA/edit?usp=sharing
         // we need to check if db is in A3 or A4.
-        io.supertokens.pluginInterface.useridmapping.UserIdMapping mapping = getUserIdMapping(tenantIdentifier, main,
-                userId,
-                UserIdType.ANY);
+        io.supertokens.pluginInterface.useridmapping.UserIdMapping mapping = getUserIdMapping(appIdentifier,
+                userId, UserIdType.ANY);
```

Comment:
so this may cause issues cause we want to find the user id mapping across all storages for the appIdentifier, and not just the one in the input appIdentifier object.

This seems to be an exception case cause we are specifically using UserIdType.ANY here and not the userIdType from the input